File: algorithms/bubble-sort/bubble-sort.rb
def bubble_sort(arr)
  arr.each_with_index do |_ , index|
    arr.each_with_index do |_, idx|
      if !arr[idx + 1].nil? && arr[idx] > arr[idx + 1]
        arr[idx + 1], arr[idx] = arr[idx], arr[idx + 1]
      end
    end
  end

  arr
end

puts bubble_sort([3,2,7,1,9])

File: algorithms/bubble-sort/bubble-sort.py
def bubble_sort(arr):
    for index, val in enumerate(arr):
        for idx, v in enumerate(arr):
            if len(arr) > (idx + 1) and arr[idx] > arr[idx + 1]:
                arr[idx + 1], arr[idx] = arr[idx], arr[idx + 1]

    print(arr)

bubble_sort([6,2,4,1,9,14,10])


